src/modules/tasks.js

```diff
@@ -0,0 +1,103 @@
+import Action from './action.js';
+import dragDrop from './drag.js';
+
+const action = new Action();
+
+export default class Tasks {
+  constructor() {
+    this.tasksArray = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  saveAndRender = () => {
+    dragDrop(document.querySelectorAll('.todo-task'));
+    localStorage.setItem('tasks', JSON.stringify(this.tasksArray));
+    this.populateList();
+  }
```

Comment:
- [ ] Good job @jonathanmunamire, I'm impressed. Only one thing left. Your task index should start from one and reflect the length of the array, task one should have an `index=1` instead of `0`, and the same thing for task two and all the others. if you have 5 tasks the last task should have `index=5`

![your-t](https://user-images.githubusercontent.com/69077061/199722127-8505522b-ad6b-4972-a103-bf4b5292041b.PNG)